x-pack/plugins/security_solution/public/common/links/types.ts

```diff
@@ -53,24 +72,33 @@ export interface LinkItem {
    * Only required for pages that are displayed inside a landing page.
    */
   landingImage?: string;
-  isBeta?: boolean;
+  /**
+   * Minimum license required to enable the link
+   */
   licenseType?: LicenseType;
+  /**
+   * Nested links
+   */
   links?: LinkItem[];
+  /**
+   * Link path relative to security root
+   */
   path: string;
+  /**
+   * Disables link in the side navigation. Defaults to false.
+   */
+  sideNavDisabled?: boolean;
+  /**
+   * Enables link in the side navigation. Defaults to false.
+   */
```

Comment:
First, thank you very much for adding comments. It is very helpful. 

I think this comment is wrong. `skipUrlState` disabled the query string in the URL.